src/js/components/Drop/__tests__/__snapshots__/Drop-test.js.snap

```diff
@@ -2044,7 +2044,7 @@ exports[`Drop resize 1`] = `
 <div
   class="c0 c1"
   id="drop-node"
-  style="left: 0px; width: 0.1px; top: 0px; max-height: 1000px;"
+  style="left: 0px; top: 0px; max-height: 1000px; width: 0.1px;"
```

Comment:
Does it make a difference having "width" located at the end of the inline CSS?

Reply:
nope. I don't think it makes a difference